src/Components/PostCard.js

```diff
@@ -0,0 +1,29 @@
+import data from '../Services/static-posts.json';
+import { CardArticle } from './StyledPostArticle';
+import { CardItem } from './StyledPostItem';
+import { CardAdresse } from './StyledPostAdresse';
+import { CardUserName } from './StyledPostName';
+
+export default function Posts() {
+	return (
+		<CardArticle>
+			<CardItem>
+				<CardUserName>
```

Comment:
Instead of using a `span` for `CardUserName`, you could simply use a `p` element and then there is no need for another nested p here.
Besides, you it is not allowed to nest a p element inside a span, [check it out here](https://caninclude.glitch.me/caninclude?child=p&parent=span).